### PR TITLE
ACS-8756: Adding config to handle ACS service accounts

### PR DESCRIPTION
**Pull request by mpichura**

### CODE REVIEW

**Review by krdabrowski — APPROVED** [review]
Nice job!

**Review by gionn — APPROVED** [review]

File: distribution/src/main/resources/docker-compose/docker-compose-w-keycloak.yml [hunk 1]
```diff
@@ -173,12 +172,12 @@ services:
...
-      AUTH_PROVIDERS_ALFRESCO_CLIENTSECRET:
-      AUTH_PROVIDERS_ALFRESCO_GRANTTYPE: password
+      AUTH_PROVIDERS_ALFRESCO_GRANTTYPE: client_credentials
       AUTH_PROVIDERS_ALFRESCO_TOKENURI: http://auth-provider:8080/auth/realms/alfresco/protocol/openid-connect/token
-      AUTH_PROVIDERS_ALFRESCO_USERNAME: admin
-      AUTH_PROVIDERS_ALFRESCO_PASSWORD: admin
+      AUTH_PROVIDERS_ALFRESCO_CLIENTID: hxi-connector
+      AUTH_PROVIDERS_ALFRESCO_CLIENTSECRET: xNnSJ6sJzQE7iHChE2pn35kHOhcACt6h
+      AUTH_PROVIDERS_ALFRESCO_SCOPE: openid email profile
+      AUTH_PROVIDERS_HYLANDEXPERIENCE_TOKENURI: http://hxinsight-mock:8080/token
```
[the remark this answers is not in the document]
Comment (mpichura) [reply distribution/src/main/resources/docker-compose/docker-compose-w-keycloak.yml]: Removed, thanks.

File: hxinsight-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-hxinsight-extension/alfresco-global.properties [hunk 2]
```diff
@@ -4,6 +4,8 @@
 ## alfresco/tomcat/shared/classes/alfresco-global.properties.
 ##
 
+serviceaccount.role.service-account-hxi-connector=ROLE_ADMIN_SERVICE_ACCOUNT
```
[the remark this answers is not in the document]
Comment (mpichura) [reply hxinsight-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-hxinsight-extension/alfresco-global.properties]: Works with empty value, but I needed to set it in the `docker-compose` file.

**Pull request merged**